lusSTR/wrappers/snps_convert.py

```diff
@@ -0,0 +1,138 @@
+import json
+import os
```

Comment:
Probably want a header in this file for sake of consistency. DHS copyright claim for this particulay code begins 2023, so that would be the appropriate as the year.